openfl/utilities/optimizers/numpy/yogi_optimizer.py

```diff
@@ -79,7 +79,8 @@ def _update_second_moment(self, grad_name: str, grad: np.ndarray) -> None:
         """
         sign = np.sign(grad**2 - self.grads_second_moment[grad_name])
         self.grads_second_moment[grad_name] = (
-            self.beta_2 * self.grads_second_moment[grad_name] + (1.0 - self.beta_2) * sign * grad**2
+            self.beta_2 * self.grads_second_moment[grad_name]
+            + (1.0 - self.beta_2) * sign * grad**2
```

Comment:
This looks like pure formatting, which isn't related to the main topic of the PR (fixing the vulnerability detected by Coverity).

Reply:
yeah its failed in code check format. will revert the commit